index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Capstone Project</title>
+    <link rel="stylesheet" href="styles.css">
+    <script defer src="script.js"></script>
+</head>
+<body>
+    <header class="home-header"> 
+        <ul class="nav">
+            <li><a href="#"><img src="images/facebook.png" alt="facebook logo" width="25px" height="25px"></a></li>
+            <li><a href="#"><img src="images/twitter.png" alt="twitter logo" width="25px" height="25px"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+        <nav class="nav-container">
+            <a href="index.html" id="logo-link"><img src="images/logo.png" alt="uacademy logo" id="logo"></a>
+            <ul class="navbar">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a class="campaign" href="#">CC Campaign</a></li>
+            </ul>
+        </nav>
+        <a href="#" class="toggle-btn">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </a>
+        <div id="hamburger-menu">
+            <span class="close-btn">&times;</span>
+            <ul class="hamburger-menu">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+      </div>
+    </header>
+    <section id="home">
+        <h3>"Hello! Sharing world"</h3>
+        <h1>Udemy Online Courses 2023</h1>
+        <p class="course-description">One of the best web development course believing in the value of improvement via learning, creating new opportunities for people from over 180 countries is taking place in February</p>
+        <div class="time">2023.02.6(MON)~7(TUS)</div>
+        <p class="course-location">@San Fransisco, Web development and more</p>
+    </section>
+    <main id="about">
+        <div class="main-program">
+            <h4>Main Program</h4>
```

Comment:
Hello @Alfredbis29. Thanks for your suggestions. 
- I have deployed my project using the main branch which is the reason. When I merge my changes it will be ok. Because after it is approved I will delete this branch.
- Can you please suggest how to add overlay color and text? I have an image, not a background image can I do it in img tag.

Reply:
Hello @Muhammad0602 , I hope you are doing great , I can recommend you to follow this :

Wrap the img tag inside a container element, such as a div.
Give the container a relative positioning, so that you can use absolute positioning for the overlay elements.
Create a new div for the overlay, and give it an absolute position, with a background color and some text.
Adjust the positioning and styling of the overlay as needed to achieve the desired effect.

If you need more explanation follow this [this video](https://www.youtube.com/watch?v=Os9kYg6PF-8)

Reply:
About deploying use the documentation I shared with you :wink:

Reply:
Hello @ Alfredbis29. Your video doesn't work. I think there are some problems with it. But I am confused should I change my deployment branch. 
You know as I said it is not working because my deployment branch (the main) is empty right now. When I will merge my changes it will be ok. I mean the LIve Demo is working but it's just showing the main branch not the feature branch in which I am currently working.

Reply:
and Thanks a lot for the suggestion